### ISSUE TITLE
MWYHL Mind What You Have Learned - lost apprentice is suspending tests even on 0 side

### ISSUE THREAD

**Issue by Gergall** [PlayersCommittee/gemp-swccg-public#726]
- [ ] Did you search through the list of **open issues** _first_ to ensure that there is not already an report for this issue?
Yes

## **Describe the bug**
MWYHL has special text on the 7-side that makes Jedi Tests suspended (not lost) when the apprentice leaves table.  But on the 0 side, if the apprentice leaves table, their Jedi Tests should be lost.  This is true even though the Jedi Tests are on table and not on the apprentice.  It's in the rulebook (AR Page 168).

## **To Reproduce**
1. Start Jedi Testing and complete at least 1 test
2. Have the apprentice leave table in some way (forfeited, Set For Stun, etc).
3. Their completed Jedi Tests on table should be lost, but they get suspended.

## **Game Replay ID:**
https://gemp.starwarsccg.org/gemp-swccg/game.html?replayId=Ericobasso$8xw70unuh8k9xt18
Dark's Turn 6 battle phase, Light forfeits Daughter Of Skywalker.

## **Expected behavior**
MWYHL 0 Side: Apprentice leaves table, their tests are lost
MWYHL 7 Side: Apprentice leaves table, their tests are suspended

## **Screenshots**
![image](https://github.com/PlayersCommittee/gemp-swccg-public/assets/6243118/dfe64c26-ce51-4562-921b-ead3e2da2695)

## **Additional context**
Probably a very old bug that never got noticed before. Let me know if any issues / rules questions about this.

**Comment by Luhks** [comment]
I played a game with MWHYL in which Daughter of Skywalker was lost after completing three Jedi tests.  The Jedi tests were not lost and they were not even suspended. I could send the replay if you want (but the game was one of my most humiliating losses ever).

**Comment by Gergall** [comment]
Yeah important clarification is that the tests are not being suspended.  I described the issue incorrectly.

The issue is that the tests remain on table.  There doesn't appear to be anything coded to make them lost in this scenario.
